docs/website/content/docs/v0.1/Configuration/servers.md

```diff
@@ -85,3 +85,27 @@ You should never change an accepted `Server` to be _not_ accepted while it is in
 Because servers which are not accepted will not be modified, if a server which
 _was_ accepted is changed to _not_ accepted, the disk will _not_ be wiped upon
 its exit.
+
+## IPMI
+
+Sidero can use IPMI information to control `Server` power state, reboot servers and set boot order.
+
+IMPI connection information can be set in the `Server` spec after initial registration:
+
+```yaml
+apiVersion: metal.sidero.dev/v1alpha1
+kind: ServerClass
```

Comment:
I think you mean `kind: Server` here, as `ServerClass` does not have `bmc` according to `kubectl explain serverclass.spec`

Reply:
Yep, this is correct. Good catch.

Reply:
thanks for catching that!